### PR TITLE
Select2 and Rules updates

### PR DESCRIPTION
**Pull request by jamesjacobs**
This branch contains the following changes:

1. Updated select2 examples to include the correct way to include placeholders.
2. Update Rule pattern markup to reflect recent changes

### CODE REVIEW

File: dist/css/pulsar.css [hunk 1]
```diff
@@ -3173,7 +3173,7 @@ body {
   text-align: center;
```
Comment (Stanton) [inline dist/css/pulsar.css:3173]: Why are the /dist files here, thought we had that in .gitignore... https://github.com/jadu/pulsar/blob/develop/.gitignore#L17

File: views/forms/select2/code.html.twig [hunk 2]
```diff
@@ -7,8 +7,20 @@
...
 
+    <p>You can find more information about the Jadu branch of selectWoo, such as screen reader coverage and known bugs on the <a href="https://github.com/jadu/selectWoo">Jadu/selectWoo GitHub repo</a></p>
+
+    {{ docs.heading('h3', 'package.json') }}
+
+    <p>The following should be included in your product’s <code>package.json</code> file.</p>
+
+    <pre><code lang="javascript">"dependencies": {
+    ...
+    "select2": "github:jadu/selectWoo#jadu-1.1.3",
```
Comment (Stanton) [inline views/forms/select2/code.html.twig:20]: Should this not be pointing to master rather than a branch?

**Review by Stanton — CHANGES_REQUESTED** [review]

File: dist/css/pulsar.css (hunk 1 above)
Comment (jamesjacobs) [reply dist/css/pulsar.css:3173]: You're linking pulsar there, we've always committed compiled styles to the docs.

**Comment by Stanton** [comment]
Oh aye, disregard!

File: views/forms/select2/code.html.twig (hunk 2 above)
Comment (jamesjacobs) [reply views/forms/select2/code.html.twig:20]: It's pointing to the latest release, jadu-1.1.3.

**Review by Stanton — APPROVED** [review]

**Pull request merged**